checkpoint_ngfw/icon_checkpoint_ngfw/actions/add_access_rule/action.py

```diff
@@ -0,0 +1,52 @@
+import komand
+from .schema import AddAccessRuleInput, AddAccessRuleOutput, Input, Output, Component
+# Custom imports below
+import requests
+
+
+class AddAccessRule(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='add_access_rule',
+                description=Component.DESCRIPTION,
+                input=AddAccessRuleInput(),
+                output=AddAccessRuleOutput())
+
+    def run(self, params={}):
+        url = f"https://{self.connection.server_ip}:{self.connection.server_port}/web_api/add-access-rule"
+
+        # Position will accept an int or string like "top" or "bottom"
+        position_string = params.get(Input.POSITION)
+        try:
+            position = int(position_string)
+        except Exception:
+            position = position_string
+
+        payload = {
+          "layer": params.get(Input.LAYER),
+          "position": position,
+          "name": params.get(Input.NAME),
+          "action": params.get(Input.ACTION),
+        }
+
+        services = params.get(Input.LIST_OF_SERVICES)
+        if services and len(services) > 0:  # Make sure it's not a blank list
```

Comment:
Trying to avoid this: 
```
Traceback (most recent call last):
  File "/Users/jmcadams/Library/Preferences/PyCharm2019.3/scratches/scratch.py", line 1, in <module>
    print(len(None))
TypeError: object of type 'NoneType' has no len()
```
It's not a required value.

Reply:
Nevermind...missed the preceding comment.